config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get '/users' , to: 'users#index';
+  get '/users/:id' , to: 'users#show';
+  get '/users/:id/posts' , to: 'posts#index';
+  get '/users/:id/posts/:id' , to: 'posts#show';
```

Comment:
- [ ] We can rewrite our routes in a more RESTFUL way to achieve the desired URLs mentioned in the project requirements. When I run `rails routes` the result of the URLs I get do not match the ones provided in the project requirements. See below:

 | **URLs from project requirements** | **URLs from your project** |
 | --- | --- |
 | ![image](https://user-images.githubusercontent.com/45002654/149008687-e33ca00d-89df-490d-b203-601bc363d73f.png) | ![image](https://user-images.githubusercontent.com/45002654/149008523-68d48a91-4a7e-4efa-afe7-f6ebe772dede.png) |
 
 Please consider rewriting the routes as below:
 
 ```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```